shark/shark_importer.py

```diff
@@ -809,6 +809,8 @@ def save_mlir(
         dir = os.path.join(".", "shark_tmp")
     mlir_path = os.path.join(dir, model_name_mlir)
     print(f"saving {model_name_mlir} to {dir}")
+    if not os.path.exists(dir):
```

Comment:
you can use os.mkdirs here because you want to make all folders in dir that don't exist

Reply:
Fixed. PTAL